api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -326,55 +366,139 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
+	 * 
+	 * @param uuid
+	 * @return concept name or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
+	 */
 	public ConceptName getConceptNameByUuid(String uuid);
 	
+	
+	/**
+	 * Get ConceptSet by its UUID
+	 * 
+	 * @param uuid
+	 * @return concept set or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
+	 */
 	public ConceptSet getConceptSetByUuid(String uuid);
 	
+	/**
+	 * Get ConceptSource by its UUID
+	 * 
+	 * @param uuid
+	 * @return concept source or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
+	 */
 	public ConceptSource getConceptSourceByUuid(String uuid);
 	
 	/**
+	 * Get ConceptDatatype by its UUID
+	 * 
 	 * @param uuid
 	 * @return concept data type or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
 	 */
 	public ConceptDatatype getConceptDatatypeByUuid(String uuid);
 	
 	/**
+	 * Get ConceptNumeric by its UUID
+	 * 
 	 * @param uuid
 	 * @return concept numeric or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
 	 */
 	public ConceptNumeric getConceptNumericByUuid(String uuid);
 	
 	/**
+	 * Get ConceptProposal by its UUID
+	 * 
 	 * @param uuid
 	 * @return concept proposal or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
 	 */
 	public ConceptProposal getConceptProposalByUuid(String uuid);
 	
 	/**
+	 * Get Drug by its UUID
+	 * 
 	 * @param uuid
-	 * @return drug or null
+	 * @return drug of null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
 	 */
 	public Drug getDrugByUuid(String uuid);
 	
+	/**
+	 * Get Drug Ingredient by its UUID
+	 * 
+	 * @param uuid the uuid for the drug ingredient to get
+	 * @return the drug ingredient if found, else null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
+	 */
 	public DrugIngredient getDrugIngredientByUuid(String uuid);
 	
 	public Map<Integer, String> getConceptUuids();
 	
+	/**
+	 * Get ConceptDescription by its UUID
+	 *
+	 * @param uuid the uuid for the concept description to get
+	 * @return concept description or null
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
+	 */	
 	public ConceptDescription getConceptDescriptionByUuid(String uuid);
 	
+	/**
+	 * Get a ConceptSource by its unique id.
+	 *
+	 * @param uuid the uuid for the concept name tag to get
+	 * @return the conceptNameTag with a matching uuid
+	 * @see Concept#setPreferredName(ConceptName)
+	 * @see Concept#setFullySpecifiedName(ConceptName)
+	 * @see Concept#setShortName(ConceptName)
+	 * @should find object given valid uuid
+	 * @should return null if no object found with given uuid
+	 */
```

Comment:
Since this method is well documented in the `ConceptService` interface, don't you wanna reference to that documentation with the help of `@see` javadoc attribute just like its done in other places in this class?